i18n/spec/utilSpec.js

```diff
@@ -1,16 +0,0 @@
-'use strict';
-
-var util = require('../src/util.js');
-
-describe('findLocaleId', function() {
-  it('should find localeId', function() {
-    expect(util.findLocaleId('', 'num')).toBeUndefined();
-    expect(util.findLocaleId('aa', 'datetime')).toBeUndefined();
-    expect(util.findLocaleId('aa', 'randomType')).toBeUndefined();
-    expect(util.findLocaleId('NumberFormatSymbols_en', 'datetime')).toBeUndefined();
-    expect(util.findLocaleId('DateTimeSymbols_en', 'num')).toBeUndefined();
-
-    expect(util.findLocaleId('DateTimeSymbols_en', 'datetime')).toBe('en');
-    expect(util.findLocaleId('NumberFormatSymbols_en_US', 'num')).toBe('en_US');
-  });
-});
```

Comment:
OK, done

On Tue, Aug 2, 2016 at 3:31 PM, Georgios Kalpakas <notifications@github.com>
wrote:

> In i18n/spec/utilSpec.js
> <https://github.com/angular/angular.js/pull/14952#discussion_r73154184>:
>
> > -'use strict';
> > -
> > -var util = require('../src/util.js');
> > -
> > -describe('findLocaleId', function() {
> > -  it('should find localeId', function() {
> > -    expect(util.findLocaleId('', 'num')).toBeUndefined();
> > -    expect(util.findLocaleId('aa', 'datetime')).toBeUndefined();
> > -    expect(util.findLocaleId('aa', 'randomType')).toBeUndefined();
> > -    expect(util.findLocaleId('NumberFormatSymbols_en', 'datetime')).toBeUndefined();
> > -    expect(util.findLocaleId('DateTimeSymbols_en', 'num')).toBeUndefined();
> > -
> > -    expect(util.findLocaleId('DateTimeSymbols_en', 'datetime')).toBe('en');
> > -    expect(util.findLocaleId('NumberFormatSymbols_en_US', 'num')).toBe('en_US');
> > -  });
> > -});
>
> closureI18nExtractor implements (and exports) a findLocaleId (which is
> just a slightly enhanced version of util.findLocaleId. There are some
> tests for findLocaleId in closureI18nExtractorSpec, but util had some
> more tests.
> Anyway, this is minor - we didn't have the tests before and they are not
> related to this PR, so it's fine to leave things as they are now (we have
> piggybacked enough on this PR 😛)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/angular.js/pull/14952/files/a35c7b1049edc8033c0e3a6eaeeda6cd34c99e73..197c8608ca2d44fec56ef25cc44c847ad7884627#r73154184>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABrUnmnFOE5ikzqnQjJyNGgk_kWV5XUTks5qb0aZgaJpZM4JU90D>
> .
>



-- 
Michał Gołębiowski